notebooks/combined_wind_wave_surge_building_loss.ipynb

```diff
@@ -0,0 +1,303 @@
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# surge-wave building damage\n",
+    "sw_bldg_dmg = BuildingDamage(client)\n",
+    "sw_bldg_dmg.load_remote_input_dataset(\"buildings\", bldg_dataset_id)\n",
+    "sw_bldg_dmg.set_input_dataset('dfr3_mapping_set', mapping_set)\n",
+    "sw_bldg_dmg.set_parameter(\"result_name\", \"Galveston-sw-dmg\")\n",
+    "sw_bldg_dmg.set_parameter(\"hazard_type\", hazard_type)\n",
+    "sw_bldg_dmg.set_parameter(\"hazard_id\", hazard_id)\n",
+    "sw_bldg_dmg.set_parameter(\"num_cpu\", 8)\n",
+    "sw_bldg_dmg.run_analysis()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab927fe1",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Flood building damage for Galveston, TX using Hurricane Ike"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ac2c18b5",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# flood mapping\n",
+    "mapping_id = \"62fefd688a30d30dac57bbd7\"\n",
+    "mapping_set = MappingSet(fragility_service.get_mapping(mapping_id))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c0800866",
+   "metadata": {
+    "pycharm": {
+     "is_executing": true,
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# flood building damage\n",
+    "f_bldg_dmg = BuildingDamage(client)\n",
+    "f_bldg_dmg.load_remote_input_dataset(\"buildings\", bldg_dataset_id)\n",
+    "f_bldg_dmg.set_input_dataset('dfr3_mapping_set', mapping_set)\n",
+    "f_bldg_dmg.set_parameter(\"result_name\", \"Galveston-flood-dmg\")\n",
+    "f_bldg_dmg.set_parameter(\"hazard_type\", hazard_type)\n",
+    "f_bldg_dmg.set_parameter(\"hazard_id\", hazard_id)\n",
+    "f_bldg_dmg.set_parameter(\"num_cpu\", 8)\n",
+    "f_bldg_dmg.run_analysis()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e6ec441",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Combine wind, wave and surge building damage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ac3ede5e",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "surge_wave_damage = sw_bldg_dmg.get_output_dataset(\"ds_result\")\n",
+    "wind_damage = w_bldg_dmg.get_output_dataset(\"ds_result\")\n",
+    "flood_damage = f_bldg_dmg.get_output_dataset(\"ds_result\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bd87445b",
+   "metadata": {
+    "pycharm": {
+     "is_executing": true,
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Structural loss ratios for each archetype\n",
+    "structural_cost_id = \"63ff7b2b5c35c0353d5edeeb\"\n",
+    "# Content loss ratios for each damage state\n",
+    "content_cost_id = \"63ff7b04a36332563d497cff\"\n",
```

Comment:
don't forget to release these two datasets. Issue see here: https://github.com/IN-CORE/pyincore-data/issues/59